test/entity2input.test.js

```diff
@@ -46,6 +46,35 @@ dateArrayField: [Date]
       )
     })
 
+     it("should convert an entity to input when entity has entity references", async () => {
```

Comment:
this test checks outputs for 'entity2input'. there should be the same test for 'entity2type'

Reply:
We believe that already exists, here is the link : https://github.com/herbsjs/herbs2gql/blob/master/test/entity2type.test.js#L126